packaging/installer/kickstart.sh

```diff
@@ -307,6 +307,7 @@ while [ -n "${1}" ]; do
 		shift 1
 	elif [ "${1}" = "--stable-channel" ]; then
 		RELEASE_CHANNEL="stable"
+		NETDATA_INSTALLER_OPTIONS="$NETDATA_INSTALLER_OPTIONS --stable-channel"
```

Comment:
Could you make this so that it doesn't add an extra space when `$NETDATA_INSTALLER_OPTIONS` is empty?

Reply:
This is harmless in bash and spaces are collapsed. I don't think it justifies the extra fluff. Also, https://github.com/netdata/netdata/blob/master/packaging/installer/kickstart-static64.sh#L193 seems to do the same.